server/test/repository/ApplicationRepositoryTest.java

```diff
@@ -10,22 +10,27 @@
 import models.DisplayMode;
 import models.LifecycleStage;
 import models.Program;
+import models.Version;
 import org.junit.Before;
 import org.junit.Test;
 import services.Path;
 
 public class ApplicationRepositoryTest extends ResetPostgres {
   private ApplicationRepository repo;
 
+  private Version draftVersion;
+
   @Before
-  public void setupApplicantRepository() {
+  public void setUp() {
     repo = instanceOf(ApplicationRepository.class);
+    draftVersion = new Version();
```

Comment:
Passed in an explicit `LifecycleStage`. I decided not to use `VersionRepository` since that felt like mixing layers of abstraction (i.e. there are no currently no dependencies from ApplicationRepository to VersionRepository).